### ISSUE TITLE
Error message on first install

### ISSUE THREAD

**Issue by Luensche** [Project60/org.project60.sepa#686]
I am trying to install the extension and I get the following error message:

`Deprecated: Required parameter $template_id follows optional parameter $send in /var/www/html/wp-content/uploads/civicrm/ext/org.project60.sepa/CRM/Sepa/Page/SepaMandatePdf.php on line 153 Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-content/uploads/civicrm/ext/org.project60.sepa/CRM/Sepa/Page/SepaMandatePdf.php:153) in /var/www/html/wp-content/plugins/civicrm/civicrm/CRM/Utils/System/Base.php on line 898 `
CiviCRM 5.65.2
CiviSEPA 1.9.1

**Comment by xavi-xaloc** [comment]
I'm also getting an error on first install that I assume is basically the same: 
`Deprecated function: Optional parameter $send declared before required parameter $template_id is implicitly treated as a required parameter in require_once() (line 166 of vendor/civicrm/civicrm-core/CRM/Core/ClassLoader.php)`.
CiviCRM 5.69.5
CiviSEPA 1.9.1

**Comment by bjendres** [comment]
Also: does it still work, or is there some sort of functional failure?

**Comment by xavi-xaloc** [comment]
No, it does'nt work, I get the message `DB Error: no such field` and I'm not able to create a mandate. 
We are using PHP 8.1.8

**Comment by peth-systopia** [comment]
(SYSTOPIA is maintaining this extension.)

**Comment by dontub** [comment]
Was fixed by https://github.com/Project60/org.project60.sepa/pull/679.
